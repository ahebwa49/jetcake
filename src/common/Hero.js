import React from "react";

const Hero = () => {
  return (
    <section className="hero">
      <div className="hero-content">
        <h1 className="hero-title">JOBS FROM TOP TECH COMPANIES</h1>

        <h2 className="hero-subtitle">
          Silicon Valley CTOs will teach you how to work with companies in the
          Silicon Valley and beyond
        </h2>

        <button type="button" className="hero-button">
          APPLY
        </button>
      </div>
    </section>
  );
};
export default Hero;
