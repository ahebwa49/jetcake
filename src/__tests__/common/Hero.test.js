import React from "react";
import configureStore from "redux-mock-store";
import { shallow, mount } from "enzyme";
import { Provider } from "react-redux";
import toJson from "enzyme-to-json";

import Hero from "../../common/Hero";

const mockStore = configureStore([]);

let component;

component = shallow(<Hero />);

test("EditProfile component renders without crashing", () => {
  expect(toJson(component)).toMatchSnapshot();
});
