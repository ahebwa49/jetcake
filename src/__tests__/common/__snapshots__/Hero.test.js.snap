// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditProfile component renders without crashing 1`] = `
<section
  className="hero"
>
  <div
    className="hero-content"
  >
    <h1
      className="hero-title"
    >
      JOBS FROM TOP TECH COMPANIES
    </h1>
    <h2
      className="hero-subtitle"
    >
      Silicon Valley CTOs will teach you how to work with companies in the Silicon Valley and beyond
    </h2>
    <button
      className="hero-button"
      type="button"
    >
      APPLY
    </button>
  </div>
</section>
`;
